packages/browser/src/client/popup.ts

```diff
@@ -0,0 +1,22 @@
+const showPopupWarning = (name: string) => (...params: any[]) => {
+  const formatedParams = params.map(p => JSON.stringify(p)).join(', ')
+
+  console.warn(`Vitest encountered a \`${name}\` call with parameters: ${
+      formatedParams}. Vitest cannot handle \`${
+      name}\` by default in the browser, so it returned null. If you need to handle this case, you can mock the \`${
+      name}\` call yourself like this:
+
+\`\`\`
+vi.spyOn(window, '${name}').mockReturnValue('your value')
+${name}(${formatedParams})
+expect(${name}).toHaveBeenCalled()
+\`\`\``)
+  return null
+}
+
+export const setupPopupSpy = () => {
+  globalThis.alert = showPopupWarning('alert')
+  // @ts-expect-error mocking native popup apis
+  globalThis.confirm = showPopupWarning('confirm')
```

Comment:
`confirm` always returns `boolean`, so you should update the return value here instead of suppressing typescript error

Reply:
Yea, tried that first, but thought returning false may be wrong. Let me change that, I think it's better.